src/wp-includes/blocks.php

```diff
@@ -375,6 +375,22 @@ function get_block_metadata_i18n_schema() {
 	return $i18n_block_schema;
 }
 
+/**
+ * Registers a block metadata collection.
+ *
+ * This function allows core and third-party plugins to register their block metadata
+ * collections in a centralized location. Registering collections can improve performance
+ * by avoiding multiple reads from the filesystem.
+ *
+ * @since 6.X.0
```

Comment:
Can you please replace these with `6.7.0` throughout the PR (hoping that we can still land this before next Tuesday)?